ci/test/00_setup_env_arm.sh

```diff
@@ -26,4 +26,4 @@ export RUN_INTEGRATION_TESTS=false
 export GOAL="install"
 # -Wno-psabi is to disable ABI warnings: "note: parameter passing for argument of type ... changed in GCC 7.1"
 # This could be removed once the ABI change warning does not show up by default
-export BITCOIN_CONFIG="--enable-reduce-exports --enable-suppress-external-warnings --enable-werror CXXFLAGS=-Wno-psabi"
+export BITCOIN_CONFIG="--enable-glibc-back-compat --enable-reduce-exports --enable-suppress-external-warnings --enable-werror CXXFLAGS=-Wno-psabi"
```

Comment:
you need only `glic-back-compat` for gitian builds, why does it enabled now for all CI?

yes, it should be tested somehow, so, not good to keep it "only for gitian", but still better to keep it only for _some_ CI but not all of them.

only ARM or only cxx200 is enough IMO

Reply:
We're enabling it for all CI builds as https://github.com/dashpay/dash/pull/5355 backports changes that run minimum glibc symbol checks on _all_ Linux builds.